core/src/main/scala/chisel3/probe/package.scala

```diff
@@ -83,7 +93,7 @@ package object probe extends SourceInfoDoc {
   /** Override existing driver of a writable probe. */
   def force(clock: Clock, cond: Bool, probe: Data, value: Data)(implicit sourceInfo: SourceInfo): Unit = {
     requireHasWritableProbeTypeModifier(probe, "Cannot force a non-writable Probe.")
-    pushCommand(ProbeForce(sourceInfo, clock.ref, cond.ref, probe.ref, value.ref))
+    pushCommand(ProbeForce(sourceInfo, clock.ref, cond.ref, probe.ref, padData(value, probe.width.get).ref))
```

Comment:
What happens if the width of a `Probe` itself is unknown?

Reply:
Repeating from the conversation on slack, firtool errors out if the probe width does not eventually compute to the force value width. So I have decided for now to just error if either of these widths are unknown, though for future purposes this may be too strict.